guides/subscriptions.md

```diff
@@ -37,6 +37,25 @@ line:
 ]
 ```
 
+in phoenix 1.4, the supervisor children are mounted like so
+
+```elixir
+# List all child processes to be supervised
+    children = [
+      # Start the Ecto repository
+      MyAppWeb.Repo,
+      # Start the endpoint when the application starts
+      MyAppWeb.Endpoint,
+      {Absinthe.Subscription, [MyAppWeb.Endpoint]}
+    ]
+
+    # See https://hexdocs.pm/elixir/Supervisor.html
+    # for other strategies and supported options
+    opts = [strategy: :one_for_one, name: Blinq.Supervisor]
```

Comment:
What is the `Blinq.Supervisor`? I fear it may confuse readers of the guide

Reply:
Agreed. Maybe `MyApp`, as we use frequently?

Reply:
I made the change as requested. that was an accidental copy over from the POC I put together